src/pages/Profile.jsx

```diff
@@ -145,7 +145,9 @@ const Profile = () => {
           width: isSmallScreen ? '100%' : 'auto'
         }}
       >
-        <Typography sx={{ fontWeight: 'bold', fontSize: '18px' }}>My Profile</Typography>
+        <Typography variant="h1" sx={{ fontWeight: 'bold', fontSize: '18px' }}>
```

Comment:
This change makes this text into a proper heading without modifying the visual rendering

Reply:
Agreed. I attempted to make minimal changes to the page itself. This seems like something we can take a look at post MVP.